web/src/components/MemoContent/CodeBlock.tsx

```diff
@@ -42,7 +42,10 @@ const CodeBlock: React.FC<Props> = ({ language, content }: Props) => {
       // Skip error and use default highlighted code.
     }
 
-    return content;
+    // escape any HTML entities when rendering original content
+    return Object.assign(document.createElement("span"), {
```

Comment:
How about div

Reply:
Because of div being a block element and span an inline one? This here will set just the text content, not the HTML, to then extract an HTML content that will only have a single text node, so everything is technically within standard and working in practice; and in this sense, since we're only writing text inside the element, I would argue span is more correct, since it's a generic element designed specifically for text. Although a div would work the exact same way.
Would you like this changed anyway?